pom.xml

```diff
@@ -221,12 +223,12 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-assembly-plugin</artifactId>
-          <version>2.5.4</version>
+          <version>3.3.0</version>
```

Comment:
This wasn't causing issues, but I don't think it hurts to keep this up-to-date for the best compatiblility.